### PR TITLE
`ACTOR_FLAG_ENKUSA_CUT` -> `ACTOR_FLAG_GRASS_DESTROYED`

### PR DESCRIPTION
**Pull request by fig02**
1) Would rather say "bush" than "ENKUSA"
2) "Cut" implies being destroyed by a sword specifically when it can be destroyed by anything (a bomb, for example)

### CODE REVIEW

**Comment by mzxrules** [comment]
My main point of disagreement is that EnKusa is called grass, both as a literal translation and in various dialogs in the game script.

**Comment by fig02** [comment]
changed

**Review by mzxrules — APPROVED** [review]

File: include/z64actor.h [hunk 1]
```diff
@@ -146,8 +146,9 @@ typedef struct ActorShape {
 //
 #define ACTOR_FLAG_10 (1 << 10)
 
-//
-#define ACTOR_FLAG_ENKUSA_CUT (1 << 11)
+// A grass bushel (EN_KUSA) has been destroyed.
```
Comment (Dragorn421) [inline include/z64actor.h:149]: "bushel"?

Comment (fig02) [reply include/z64actor.h:149]: Yes. You cant just say "a grass", it means like a bunch of grass clumped together, which is kinda like what the en_kusa actor is

**Pull request merged**